doc/reference/pipeline_spec.md

```diff
@@ -429,24 +457,29 @@ Union inputs take the union of other inputs. For example:
 |        |        | buzz            |
 ```
 
-Notice that union inputs, do not take a name and maintain the names of the
-sub-inputs. In the above example you would see files under
+The union inputs do not take a name and maintain the names of the
+sub-inputs. In the example above, you would see files under
 `/pfs/inputA/...` or `/pfs/inputB/...`, but never both at the same time.
-This can be annoying to write code for since the first thing your code
-needs to do is figure out which input directory is present. As of 1.5.3
-the recommended way to fix this is to give your inputs the same `Name`,
-that way your code only needs to handle data being present in that
-directory. This, of course, only works if your code doesn't need to be
+When you write code to address this behavior, make sure that
+your code first determines which input directory is present. Starting
+with Pachyderm 1.5.3, we recommend that you give your inputs the
+same `Name`. That way your code only needs to handle data being present
+in that directory. This only works if your code does not need to be
 aware of which of the underlying inputs the data comes from.
 
-`input.union` is an array of inputs to union, note that these need not be
-`pfs` inputs, they can also be `union` and `cross` inputs. Although there's no
-reason to take a union of unions since union is associative.
+`input.union` is an array of inputs to a union. The inputs do not have to be
```

Comment:
union is being used as a verb here, not a noun so remove "a" or perhaps we should not use that verb, although I dont have an alternative in mind.

Reply:
combine?

Reply:
yeah combine is great.